runtime/gc_modron_startup/mmparse.cpp

```diff
@@ -1261,6 +1261,19 @@ gcParseXXArguments(J9JavaVM *vm)
 		}
 	}
 
+	{
+		IDATA adaptiveGCThreadingIndex = FIND_ARG_IN_VMARGS(EXACT_MATCH, "-XX:+AdaptiveGCThreading", NULL);
+		IDATA noAdaptiveGCThreadingIndex = FIND_ARG_IN_VMARGS(EXACT_MATCH, "-XX:-AdaptiveGCThreading", NULL);
+		if (adaptiveGCThreadingIndex != noAdaptiveGCThreadingIndex) {
+			/* At least one option is set. Find the right most one. */
+			if (adaptiveGCThreadingIndex > noAdaptiveGCThreadingIndex) {
+				extensions->adaptiveGCThreading = true;
```

Comment:
Where is `adaptiveGCThreading` declared? I don't see it in the existing code, nor in this pull request.

Reply:
This PR depends on https://github.com/eclipse/omr/pull/5830, it still has to be merged